drop-transfer/src/ws/client/mod.rs

```diff
@@ -92,6 +92,10 @@ async fn connect_to_peer(
         }
     };
 
+    state
+        .moose
+        .service_quality_transfer_batch(xfer.id().to_string(), xfer.info(), ver.into());
+
```

Comment:
This was moved from another place, could you explain the reasoning?

Reply:
this is the earliest point, in which we can get the protocol version, to provide to the event

Reply:
Oh okay, makes sense. I guess we report the version that's used for the transaction, meaning if we maintain backwards compatibility and we have v5 and v1 communicating, it's gonna report v1? Just checking as it makes sense.

Is there a possibility to report the highest supported version as well so in this case it would report both 5 and 1?

Reply:
Yes, in the case of v5 and v1 communicating, it would report v1, including highest supported version would require another revision of the tracker, but we also include the lib version in the data, so we could infer that from it

Reply:
That's right!